app/jobs/isolate_job.rb

```diff
@@ -20,6 +22,7 @@ def perform(submission)
       write
       if compile == :failure
         clean
+        call_webhooks(submission)
```

Comment:
You don't need to pass `submission` as an argument because it is available as member of this class.

Reply:
Noted, will fix